app/assets/stylesheets/govuk_publishing_components/components/_select.scss

```diff
@@ -3,3 +3,11 @@
 .gem-c-select__select--full-width {
   width: 100%;
 }
+
+// Solution to text inside selects becoming unreadable if font size in
+// the browser is increased. This is currently a problem in govuk-frontend
+.gem-c-select {
+  .govuk-select {
```

Comment:
It'd be good to catchup with @alex-ju I think you may need to cater for both cases:

1. compatibility mode turned on
2. compatibility mode turned off

I still think we could fix this in govuk-frontend...

Reply:
> I still think we could consider a fix upstream in govuk-frontend first?

Yes, if that's an option. Sorry, the impression I had was that it wasn't. Happy to have that happen, although we need a fix quickly and I didn't want to put pressure on anyone.

Reply:
> I've raised an issue in govuk-frontend but it's likely that their fix will not help on GOV.UK as we're using compatibility mode for elements and toolkit, and this issue apparently relates to the base font size.

I think I may have miscommunicated, we already output different units for the font scale depending on if the compatibility mode is set, we may be able to use similar logic to solve this issue too.

Reply:
Spoke with Andy, and we'll try for a long term fix on govuk-frontend but that shouldnt block quicker improvements here.